paasta_tools/tron_tools.py

```diff
@@ -316,6 +323,7 @@ def _get_action_config(self, action_name, action_dict):
                 )
         else:
             branch_dict = None
+        action_dict['monitoring'] = self.get_monitoring()
```

Comment:
This line feels out of place. I understand why we "need" it, but I feel like this should actually override the `get_monitoring` method in `TronActionConfig`.

Looking at the code though, it looks like this "is" how we are injecting monitoring config in other InstanceConfigs. idk, it looks like how we handle monitoring data in our instance configs isn't super tight. They may break in the future, but for now I say ship